internal/cmd/base/dev.go

```diff
@@ -29,7 +31,7 @@ func (b *Server) CreateDevDatabase(ctx context.Context, opt ...Option) error {
 	// We should only get back postgres for now, but laying the foundation for non-postgres
 	switch opts.withDialect {
 	case "":
-		b.Logger.Error("unsupported dialect. wanted: postgres, got: %v", opts.withDialect)
+		event.WriteError(ctx, op, err, event.WithInfoMsg(fmt.Sprintf("unsupported dialect. wanted: postgres, got: %v", opts.withDialect)))
```

Comment:
Probably this should be something like
`event.WithInfoMsg("unsupported dialect", "wanted", "postgres", "got", opts.withDialect`